src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-inline: 50px;
+  align-items: center;
+  position: relative;
+
+  &__logo {
+    height: 40px;
+    margin: auto 0;
+
+    &:hover {
+      border: 4px;
```

Comment:
The border property is missing a unit for the border width. It should be specified, for example, as '4px'.